Formula/imagemagick.rb

```diff
@@ -72,49 +72,149 @@ def install
       --enable-static
     ]
 
+    if build.without? "freetype"
+      args << "--without-freetype"
+    else
+      args << "--with-freetype"
+    end
+
+    if build.without? "jpeg"
+      args << "--without-jpeg"
+    else
+      args << "--with-jpeg"
+    end
+
+    if build.without? "libpng"
+      args << "--without-png"
+    else
+      args << "--with-png"
+    end
+
+    if build.without? "libtiff"
+      args << "--without-tiff"
+    else
+      args << "--with-tiff"
+    end
+
+    if build.without? "magick-plus-plus"
+      args << "--without-magick-plus-plus"
+    else
+      args << "--with-magick-plus-plus"
+    end
+
     if build.without? "modules"
       args << "--without-modules"
     else
       args << "--with-modules"
     end
 
+    if build.without? "threads"
+      args << "--without-threads"
+    else
+      args << "--with-threads"
+    end
+
+    if build.with? "fftw"
+      args << "--with-fftw"
+    else
+      args << "--without-fftw"
+    end
+
+    if build.with? "fontconfig"
+      args << "--with-fontconfig"
+    else
+      args << "--without-fontconfig"
+    end
+
+    if build.with? "ghostscript"
+      args << "--with-gslib" << "--with-gs-font-dir=#{HOMEBREW_PREFIX}/share/ghostscript/fonts"
+    else
+      args << "--without-gslib"
+    end
+
+    if build.with? "hdri"
+      args << "--enable-hdri"
+    else
+      args << "--disable-hdri"
+    end
+
+    if build.with? "liblqr"
+      args << "--with-lqr"
+    else
+      args << "--without-lqr"
+    end
+
+    if build.with? "librsvg"
+      args << "--with-rsvg"
+    else
+      args << "--without-rsvg"
+    end
+
+    if build.with? "libwmf"
+      args << "--with-wmf"
+    else
+      args << "--without-wmf"
+    end
+
+    if build.with? "little-cms"
```

Comment:
No, I've checked multiple times already and there's no option for `little-cms2`. There are env vars `LCMS2_CFLAGS` and `LCMS2_LIBS` to override `pkg-config`, otherwise nothing.